osf/external/gravy_valet/translations.py

```diff
@@ -65,6 +77,11 @@ class EphemeralAddonConfig:
     label: str
     short_name: str
     full_name: str
+    has_widget: bool = False
```

Comment:
I'd convert this to a property or exclude this from init
```suggestion
    has_widget: bool = dataclasses.field(init=False, default=False)
```

Reply:
Done